src/main/webapp/my-lists.html

```diff
@@ -59,47 +59,66 @@
-            <form class="form-inline result" id="experiences-form" display="none">
-                <label class="my-1 mr-2 sr-only" for="newExperience">Add a new experience</label>
-                <input type="text" class="form-control my-1 mr-sm-2" id="newExperience" placeholder="Add a new experience" required>
-
-                <button type="submit" class="btn btn-primary mb-2">Add</button>
-            </form>
-        </div> -->
-
-        <!-- <div class="container-fluid text-center bg-white">
-            <h1>Goals</h1>
-            <ul class="list-group" id="goals"></ul>
-
-            <form class="form-inline result" id="goals-form" display="none">
-                <label class="my-1 mr-2 sr-only" for="newGoal">Add a new goal for yourself</label>
-                <input type="text" class="form-control my-1 mr-sm-2" id="newGoal" placeholder="Add a new goal for yourself" required>
-
-                <button type="submit" class="btn btn-primary mb-2">Add</button>
-            </form>
-        </div>
-
-        <div class="bg-description-blue">
-            <div class="container-fluid text-center bg-white">
-                <h1>Progress</h1>
-                <h3>Achievements</h3>
-                <ul class="list-group" id="achievements"></ul>
-
-                <h3>Tutoring Sessions</h3>
-                <ul class="list-group" id="past-sessions"></ul>
-
-                <h3>Topics Covered</h3>
-                <ul class="list-group" id="past-topics"></ul>
+        <!-- Modal -->
+        <div class="modal fade" id="new-list-modal" tabindex="-1" role="dialog" aria-hidden="true">
+            <div class="vertical-alignment-helper">
+                <div class="modal-dialog modal-dialog-centered vertical-align-center" role="document">
+                    <div class="modal-content">
+                    <div class="modal-header">
+                        <h3 class="modal-title" id="new-list-form-title" style="display:inline-block">Create a new list</h3>
+                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
+                        </button>
+                    </div>
+                    <div class="modal-body">
+                        <form id="new-list-form" class="form-inline" action="/manage-lists" method="POST">
+                            <div class="form-group mx-sm-3 mb-2"><input name="name" class="form-control" type="text" id="list-name-input" required="" aria-required=true placeholder="List name"></div>
+                            <div class="form-group" style="float:right">
+                                <label for="list-topic-select">Topic:</label>
+                                <select name="topic" id="list-topic-select" class="form-control" onchange="checkOtherSelected(this.value)" required>
+                                    <!-- <option selected disabled>Topic</option> -->
```

Comment:
What's this comment mean?

Reply:
Sorry, this comment was supposed to be deleted. It was basically just setting the selection form to a disabled option.